x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -34,6 +38,28 @@ interface AlertsTableTGridProps {
   setRefetch: (ref: () => void) => void;
 }
 
+// TODO import EventsThContent from timelines plugin and customize a few css properties (position & padding-top)
+const EventsThContent = styled.div.attrs(({ className = '' }) => ({
```

Comment:
@XavierM Requirement here was to add an `Actions` header, so temporarily I copied the styled EventsThContent component from the styles file of the t_grid component. I was thinking to import the component instead of copying, but I was wondering if there is another way to do custom styling on elements.

Reply:
The `EventsThContent` `styled` component may be deleted when the transition to `EuiDataGrid` is complete, so it's reasonable to copy it here, as opposed to making it part of the public API of tgrid (so it can be imported).

> I was wondering if there is another way to do custom styling on elements.

The `EuiDataGridControlColumn` interface exposes `headerCellRender` to enable custom styling of header cells.  This PR is correctly using it [here](https://github.com/elastic/kibana/pull/105446/files#diff-f9d2565b68bb75d711c69a2d1c21a6e9c4ced3494068fee60845b93e4eca1895R128):

```
headerCellRender: () => {
        return <EventsThContent>Actions</EventsThContent>; // TODO: internationalization
      },
```

to provide the custom styled header. I'm not aware of another way to do it, and this looks good to me. 😄